src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.UserNotAllowed;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
Consider using `@BeforeEach` or `@BeforeAll` for initializing the `RegistrationService`. If the `RegistrationService` is stateless and its initialization doesn't depend on instance variables of the test class, `@BeforeAll` could be a better choice to initialize it only once. However, if each test requires a fresh instance of `RegistrationService`, then `@BeforeEach` is appropriate.